internal/powerbi/resource_pbix.go

```diff
@@ -46,22 +46,26 @@ func ResourcePBIX() *schema.Resource {
 			},
 			"skip_report": {
 				Type:        schema.TypeBool,
-				Description: "If true only the PBIX dataset is deployed.",
+				Description: "If true, only the PBIX dataset is deployed.",
 				Optional:    true,
 				Default:     false,
 			},
 			"report_id": {
 				Type:        schema.TypeString,
 				Description: "The ID for the report that was deployed as part of the PBIX.",
-				Optional:    true,
 				Computed:    true,
 			},
 			"dataset_id": {
 				Type:        schema.TypeString,
 				Description: "The ID for the dataset that was deployed as part of the PBIX.",
-				Optional:    true,
 				Computed:    true,
 			},
+			"rebind_dataset_id": {
+				Type:          schema.TypeString,
+				Description:   "If set, will rebind the report to the the specified dataset ID.",
+				Optional:      true,
+				ConflictsWith: []string{"parameter", "datasource"},
```

Comment:
Added conflictsWith to avoid having to deal with the question of which dataset's parmeters and datasources are being changed